### ISSUE TITLE
qytdl-1.5 failed to launch

### ISSUE THREAD

**Issue by hervyqa** [jahendrie/qytdl#7]
qytdl version. try to compiling in void linux.
```
➜  ~ xbps-query -p pkgver qytdl
qytdl-1.5_1
```
python version
```
➜  ~ python3 --version
Python 3.10.8
```
```
➜  ~ qytdl
Traceback (most recent call last):
  File "/usr/share/qytdl/src/./main.py", line 29, in <module>
    from main_window import MainWindow
  File "/usr/share/qytdl/src/main_window.py", line 17, in <module>
    class MainWindow( QMainWindow ):
  File "/usr/share/qytdl/src/main_window.py", line 19, in MainWindow
    opts = read_config()
  File "/usr/share/qytdl/src/config.py", line 157, in read_config
    read_options( opts )
  File "/usr/share/qytdl/src/config.py", line 144, in read_options
    section = cfg[ "Theme" ]
  File "/usr/lib/python3.10/configparser.py", line 964, in __getitem__
    raise KeyError(key)
KeyError: 'Theme'
```
install with source the result is also error
```
➜  qytdl git:(master) doas make install
doas (l7@studio) password: 
find . -type f -exec install -Dm 755 "{}" "/usr/share/qytdl/{}" \;
install qytdl.desktop -D /usr/share/applications/qytdl.desktop
install README -D /usr/share/doc/qytdl/README
install doc/CHANGES -D /usr/share/doc/qytdl/CHANGES
install doc/LICENSE -D /usr/share/licenses/qytdl/LICENSE
install qytdl.sh -D /usr/bin/qytdl
➜  qytdl git:(master) qytdl
Traceback (most recent call last):
  File "/usr/share/qytdl/src/./main.py", line 29, in <module>
    from main_window import MainWindow
  File "/usr/share/qytdl/src/main_window.py", line 17, in <module>
    class MainWindow( QMainWindow ):
  File "/usr/share/qytdl/src/main_window.py", line 19, in MainWindow
    opts = read_config()
  File "/usr/share/qytdl/src/config.py", line 157, in read_config
    read_options( opts )
  File "/usr/share/qytdl/src/config.py", line 144, in read_options
    section = cfg[ "Theme" ]
  File "/usr/lib/python3.10/configparser.py", line 964, in __getitem__
    raise KeyError(key)
KeyError: 'Theme'
➜  qytdl git:(master) 
```

**Comment by jahendrie** [comment]
Not quite sure why it would choke on that section.  Try deleting the config (default location ~/.config/qytdl/qytdl.cfg) and re-run the program so that it regenerates the default config.  Push comes to shove, I added a --no-config option so that it can skip checking the config file entirely if need be.

**Comment by hervyqa** [comment]
still error.
![Screenshot_20221102_214758](https://user-images.githubusercontent.com/45872139/199521071-d15ba0a0-5b93-415c-8e88-7d57b1926732.png)

**Comment by jahendrie** [comment]
Which version are you using?  If it isn't 1.6, try updating it.

**Comment by hervyqa** [comment]
solved with 1.6. thank you so much..
